app/src/main/res/values-DE/strings.xml

```diff
@@ -0,0 +1,306 @@
+
+    <string name="dialog_title_duplicate_artist">Doppelter Künstler</string>
+    <string name="dialog_msg_duplicate_artist">Künstler %1$s existiert bereits.</string>
+
+    <string name="dialog_title_choose_playlist">Wiedergabeliste auswählen</string>
+
+    <string name="dialog_title_edit_playlist">Wiedergabeliste bearbeiten</string>
+
+    <string name="dialog_title_choose_backup_content">Wählen Sie den Sicherungsinhalt</string>
+    <string name="dialog_title_choose_restore_content">Wählen Sie Inhalt wiederherstellen</string>
+    <string name="choice_preferences">Einstellungen</string>
+    <string name="choice_database">Datenbank</string>
+    <string name="choice_downloaded_songs">Heruntergeladene Lieder</string>
+    <string name="message_backup_create_success">Sicherung erfolgreich erstellt</string>
+    <string name="message_backup_create_failed">Konnte keine Sicherung erstellen</string>
+    <string name="message_restore_failed">Wiederherstellung der Sicherung fehlgeschlagen</string>
+
+    <!-- Notification -->
+    <string name="channel_name_playback">Musik-Player</string>
+    <string name="channel_name_download">Herunterladen</string>
+
+    <!-- Noun -->
+    <plurals name="song_count">
+        <item quantity="one">%d Lied</item>
+        <item quantity="other">%d Lied</item>
+    </plurals>
+    <plurals name="artist_count">
+        <item quantity="one">%d Künstler</item>
+        <item quantity="other">%d Künstler</item>
+    </plurals>
+    <plurals name="album_count">
+        <item quantity="one">%d Album</item>
+        <item quantity="other">%d Alben</item>
+    </plurals>
+    <plurals name="playlist_count">
+        <item quantity="one">%d Wiedergabeliste</item>
+        <item quantity="other">%d Wiedergabelisten</item>
+    </plurals>
+
+    <!-- Button -->
+    <string name="btn_retry">Wiederholen</string>
+    <string name="btn_play">Spielen</string>
+    <string name="btn_play_all">Alles abspielen</string>
+    <string name="btn_radio">Radio</string>
+    <string name="btn_shuffle">Shuffle</string>
+    <string name="btn_copy_stacktrace">Stapelspur kopieren</string>
+    <string name="btn_report">Bericht</string>
+    <string name="btn_report_on_github">Bericht auf GitHub</string>
+
+    <!-- Sort Menu -->
+    <string name="sort_by_create_date">Datum hinzugefügt</string>
+    <string name="sort_by_name">Name</string>
+    <string name="sort_by_artist">Künstler</string>
+    <string name="sort_by_year">Jahr</string>
+    <string name="sort_by_song_count">Anzahl der Lieder</string>
+    <string name="sort_by_length">Länge</string>
+    <string name="sort_by_play_time">Spielzeit</string>
+
+    <!-- Snackbar -->
+    <plurals name="snackbar_delete_song">
+        <item quantity="one">%d Song wurde gelöscht.</item>
+        <item quantity="other">%d Lieder wurden gelöscht.</item>
+    </plurals>
+    <plurals name="n_selected">
```

Comment:
Need one line below:
`<item quantity="one">{string when n is 1}</item>`

Reply:
> 

Sorry if I ask again stupid but which line exactly and what should be in the line.

Reply:
> Need one line below: `<item quantity="one">{string when n is 1}</item>`

I have inserted it, it should be correct now.
Thank you.